plugins/node/opentelemetry-instrumentation-aws-sdk/README.md

```diff
@@ -0,0 +1,120 @@
+# OpenTelemetry aws-sdk Instrumentation for Node.js
+
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+[component owners](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/.github/component_owners.yml): @willarmiros @NathanielRN @blumamir
+
+This module provides automatic instrumentation for [`aws-sdk` v2](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/) and [`@aws-sdk` v3](https://github.com/aws/aws-sdk-js-v3)
+
+## Installation
+
+```bash
+npm install --save opentelemetry-instrumentation-aws-sdk
+```
+
+## Usage
+For further automatic instrumentation instruction see the [@opentelemetry/instrumentation](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-instrumentation) package.
+
+```js
+const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node');
+const { registerInstrumentations } = require('@opentelemetry/instrumentation');
+const { AwsInstrumentation } = require('opentelemetry-instrumentation-aws-sdk');
+
+const traceProvider = new NodeTracerProvider();
+traceProvider.register();
+
+registerInstrumentations({
+  traceProvider,
```

Comment:
Please remove passing the traceProvider here like we did it in other samples. Instrumentations should pick up the global `TracerProvider`.